src/MusicStore/Properties/AppSettings.cs

```diff
@@ -3,5 +3,6 @@
     public class AppSettings
     {
         public string SiteTitle { get; set; }
+        public int CacheTimeoutSeconds { get; set; } = 600;
```

Comment:
If the idea is to just enable/disable cache, then probably requires a better name and bool value?